packages/@o3r/application/src/devkit/application-devtools.console.service.ts

```diff
@@ -42,16 +41,18 @@ export class ApplicationDevtoolsConsoleService implements DevtoolsServiceInterfa
   /** @inheritDoc */
   public activate() {
     const windowWithDevtools: WindowWithDevtools = window;
-    // eslint-disable-next-line no-underscore-dangle
+    // eslint-disable-next-line no-underscore-dangle -- naming convention
```

Comment:
You could add this [config](https://github.com/AmadeusITGroup/otter/pull/2346/files#diff-3d5c708300ad7948b79d8fe288d25858f6a31a2595e1921cd83e07522f8eaea9)
so you can remove these lines (44 and 46)